common.js

```diff
@@ -0,0 +1,29 @@
+/* globals omitTerms, respecConfig, $, require */
+/* exported linkCrossReferences, restrictReferences, fixIncludes */
+
```

Comment:
Thank you very much for this tip. I had pending how to include current drafts on the bibliography!